blog/2021-05-17-WritingFastConstraintStreamsSecretRecipe.adoc

```diff
@@ -0,0 +1,193 @@
+= Writing fast constraints with OptaPlanner: The secret recipe
```

Comment:
The JBake migration is almost done, but getting it to publish automatically might take a few days. LEt's get this out first and I 'll deal with merge conflicts on my end (same for Jiri's PR, I 'll merge it tomorrow)

Let's put the date on 25 or 26 May (adjust the file name) and merge(=publish) it.